src/ImageProcessorCore/Colors/ColorTransforms.cs

```diff
@@ -18,9 +19,60 @@ namespace ImageProcessorCore
     public partial struct Color
     {
         /// <summary>
+        /// Blends two colors by lightening.
+        /// </summary>
+        /// <remarks>
+        /// Selects the lighter of the backdrop and source colors.
+        /// The backdrop is replaced with the source where the backdrop is lighter; otherwise, it is left unchanged
+        /// Note: This behavior is on a channel by channel basis, i.e., this rule is applied to each of the 3 RGB color channels separately.
+        /// </remarks>
+        /// <param name="backdrop"></param>
+        /// <param name="source"></param>
+        /// <returns>
+        /// The <see cref="Color"/>.
+        /// </returns>
+        public static Color Lighten(Color backdrop, Color source)
+        {
+            var newVector = new Vector4
```

Comment:
>Pre-multiplied red w/ 50% opacity: 255, 0, 0, .5
>Post-multiplied red w/ 50% opacity: 127.5, 0, 0, .5

You're right about the calculation those values should be below in your example though. (255 is bytes in Bgra32)

    1, 0, 0, .5
    .5, 0, 0, .5

Is `Cr = (1 - αb) x Cs + αb x B(Cb, Cs)` the formula then that is applicable for both opaque and premultiplied values?

All the colours in the Image class are premultiplied when loaded from the various image formats so we should assume that everything is premultiplied.